libtransmission/ptrarray.c

```diff
@@ -29,6 +29,22 @@ void tr_ptrArrayDestruct(tr_ptrArray* p, PtrArrayForeachFunc func)
     tr_free(p->items);
 }
 
+tr_ptrArray* tr_ptrArrayNew(void)
+{
+    tr_ptrArray * p = tr_new(tr_ptrArray, 1);
+    *p = TR_PTR_ARRAY_INIT;
+    return p;
+}
+
+tr_ptrArray* tr_ptrArrayDup(tr_ptrArray* in)
```

Comment:
`tr_ptrArrayDup()` doesn't appear to be used anywhere in this PR and so should be removed